src/app/Models/Traits/CrudTrait.php

```diff
@@ -51,23 +56,42 @@ public static function getEnumValuesAsAssociativeArray($field_name)
         return $array;
     }
 
+    /**
+     * Register aditional types in doctrine schema manager for the current connection.
+     *
+     * @param Model $instance
+     * @return DB
+     */
+    public static function getConnectionWithExtraTypeMappings($instance)
```

Comment:
I see this method is always called on a model, but has the model passed as a parameter too. I don't think that makes a lot of sense, to call ```$model:: getConnectionWithExtraTypeMappings(new $model)```.

I think it's better if we remove that parameter entirely, and do ```$instance = new self;``` inside the method.

Reply:
Done.